definitions/npm/ramda_v0.21.x/flow_>=v0.23.x_<=v0.27.x/ramda_v0.21.x.js

```diff
@@ -1,642 +0,0 @@
-/* @flow */
-/* eslint-disable no-unused-vars, no-redeclare */
-
-type Transformer<A,B> = {
```

Comment:
Does it make sense to leave this old libdef in place for anyone using older versions of Flow?